framework/doc/content/source/materials/CoupledValueFunctionMaterial.md

```diff
@@ -0,0 +1,14 @@
+# CoupledValueFunctionMaterial
+
+!syntax description /Materials/CoupledValueFunctionMaterial
+
```

Comment:
Does the capability introduced in this PR have to be necessarily used from a material?